File: 7-kyu/unlucky-days/python/solution.py
from datetime import date, timedelta

def unlucky_days(year):
    d = date(year, 1, 1)
    while d.weekday() != 4:
        d += timedelta(days=1)
    ret = 0
    while d.year == year:
        d += timedelta(weeks=1)
        if d.day == 13:
            ret += 1
    return ret

File: 7-kyu/unlucky-days/javascript/solution.js
const unluckyDays = year => {
  let dayCount = 0;
  for (let i = 0; i < 12; i++) {
    const date = new Date(year, i, 13);
    if (date.getDay() === 5) {
      dayCount++;
    }
  }
  return dayCount;
};